nikola/conf.py.in

```diff
@@ -343,6 +356,25 @@ HIDDEN_TAGS = ['mathjax']
 # However, more obscure tags can be hidden from the tag index page.
 # TAGLIST_MINIMUM_POSTS = 1
 
+# A list of dictionaries specifying tags which translate to each other.
+# For example:
+#   [
+#     {'en': 'car', 'de': 'Auto'},
+#     {'en': 'window', 'fr': 'fenêtre'},
+#   ]
+# Note that tags in the same language can appear more than once, for
+# example if they have different meaning:
+#   [
+#     {'en': 'cloud', 'de': 'Wolke'},  # the fluffy thing in the sky
+#     {'en': 'cloud', 'de': 'Cloud'},  # cloud computing
```

Comment:
@Kwpolska: so do you want the examples removed/simplified to not use collisions?